### ISSUE TITLE
Python 3: ImportError: cannot import name 'encode' from 'base92' (/usr/local/lib/python3.7/dist-packages/base92/__init__.py)

### ISSUE THREAD

**Issue by comydream** [thenoviceoof/base92#9]
The module works well in Python 2, but when I try the code in Python 3, it raises this error:

**ImportError: cannot import name 'encode' from 'base92' (/usr/local/lib/python3.7/dist-packages/base92/__init__.py)**

**Comment by Grimsfury** [comment]
Same issue here. Did you find a solution?

**Comment by BrendanSimon** [comment]
I fixed the the import error my simply modifying the installed code (in my venv).

Edit file `.../lib/python3.7/site-packages/base92/__init__.py`

Change `from base92 import ...` to `from .base92 import ...`

HOWEVER, the `encode()` method does not work.  There are other Python3 issues that need to be sorted.

`basestr` does not exist in Python 3.  Need to use `str` instead.  Can use the `six` module (`six.string_types`) or other ways for Py2/3 compatibility.

After fixing that there are still more Python3 issues (`chr()`` and `ord()`).  I stopped there as it was not important, but I might have a go at porting for Python3 compatibility at some stage.

For now, I'd say Python3 is not supported :(
